scripts/claude_mcp.sh

```
@@ -0,0 +1,1733 @@
+# which is HTTP-based and hosted remotely, replacing the old deprecated npm package
+
+# Load centralized token helper
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+TOKEN_HELPER="$SCRIPT_DIR/scripts/load_tokens.sh"
+
+if [ -f "$TOKEN_HELPER" ]; then
+    echo -e "${BLUE}📋 Loading tokens using centralized helper...${NC}"
+    log_with_timestamp "Using centralized token helper: $TOKEN_HELPER"
+
+    # Source the token helper to load functions and tokens
+    source "$TOKEN_HELPER"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)